corresp:
  n_features: 500
  distance_threshold_px: 3
  distances_ratio: 0.5
  reprojection_error_px: 3
  encoding: "bgr8"

debug_sett:
  debug_images: true
  debug_matches: true
  debug_markers: false
  debug_distances: false

algorithms:
#  triangulation method can be so-called primitive or svd
  triangulation_method: "svd"
#  triangulation_method: "primitive"
